app/views/create-route.html

```diff
@@ -40,7 +41,9 @@
                               ng-click="createRoute()"
                               ng-disabled="createRouteForm.$invalid || disableInputs || !createRoute"
                               value="">Create</button>
-                          <a class="btn btn-default btn-lg" href="#" back>Cancel</a>
+                          <a class="btn btn-default btn-lg"
+                              ng-click="doneEditing = true"
```

Comment:
we should use dot notation instead of setting a top level scope var since some of these are underneath ng-ifs